### DISCUSSION TITLE
French translation group

### DISCUSSION THREAD

**Discussion by ManuSquall** [microsoft/TypeScript-Website-Localizations#108] [General]
<html>
<body>
<p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px !important; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Welcome to the french translation group!</p><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">So translating becomes easier, we are hoping that you follow some directives about how we cooperate, how to review a translation and how to translate specific terms.</p><h1 style="box-sizing: border-box; font-size: 2em; margin: 24px 0px 16px; font-weight: 600; line-height: 1.25; border-bottom: 1px solid var(--color-border-secondary); padding-bottom: 0.3em; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">How to cooperate</h1><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Before choosing any subject to translate, please check<span> </span><a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="943558532" data-permission-text="Title is private" data-url="https://github.com/microsoft/TypeScript-Website-Localizations/issues/100" data-hovercard-type="issue" data-hovercard-url="/microsoft/TypeScript-Website-Localizations/issues/100/hovercard" href="https://github.com/microsoft/TypeScript-Website-Localizations/issues/100" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none; white-space: normal;">#100</a><span> </span>for available subjects.<br style="box-sizing: border-box;">When you've found an available subject, please tell others that you're doing it by replying to this discussion with the subject that you've chosen.<br style="box-sizing: border-box;">To start translating, fork this repository and translate in another branch.<br style="box-sizing: border-box;">When done, do a pull request with your translated files. You can ask for reviews of your translation by replying to this discussion.</p><h1 style="box-sizing: border-box; font-size: 2em; margin: 24px 0px 16px; font-weight: 600; line-height: 1.25; border-bottom: 1px solid var(--color-border-secondary); padding-bottom: 0.3em; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">How to review</h1><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">When reviewing, it is mandatory to check for misspelled words and Anglicism. Some words in English can't be kept as they are. If you are not sure how to translate something, please discuss about it here or in your PR, don't start an issue.</p><h1 style="box-sizing: border-box; font-size: 2em; margin: 24px 0px 16px; font-weight: 600; line-height: 1.25; border-bottom: 1px solid var(--color-border-secondary); padding-bottom: 0.3em; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">How to translate</h1><p style="box-sizing: border-box; margin-bottom: 16px; margin-top: 0px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Some terms appear multiple times along the courses. For consistency, it is therefore preferred that we agree to each other on a common vocabulary. If you encounter a word that isn't in this list and should be, please tell us about it!</p>

English | French
-- | --
Fixits | Correction automatique
root fields | champs racine
a nightly build | une nightly build

<p style="box-sizing: border-box; margin-bottom: 0px !important; margin-top: 0px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">We are happy to have you among us and hope you'll have fun translating!</p>
</body>
</html>

**Comment by PhilippePerret** [comment]
Bonjour,
Peut-on parler en français ici ?
J'ai commencé à corriger des fautes, mais je rencontre un problème au `yarn run` de ma [pull request](https://github.com/microsoft/TypeScript-Website-Localizations/pull/236) avec des erreurs que je ne comprends pas.
Merci de votre aide.
